client/lib/wp/sync-handler/whitelist-handler.js

```diff
@@ -8,7 +8,8 @@ const debug = debugFactory( 'calypso:sync-handler:whitelist' );
 const whitelist = [
 	/^\/me\/posts$/,
 	/^\/me\/settings/,
-	/^\/sites\/[\w.]+\/posts/
+	/^\/sites\/[\w.]+\/posts/,
+	/^\/videos/
```

Comment:
I personally don't understand the implication of adding this to the whitelist. Can you explain or link me to a reference?